tasks/bundle.js

```diff
@@ -66,3 +86,52 @@ function bundle() {
 }
 
 gulp.task('bundle', bundle);
+
+const licenses = `
+# The MIT License (MIT)
+
```

Comment:
Could you just do 

Basscss | https://github.com/basscss/basscss/blob/master/LICENSE.md
---------------------------------
ACTUAL Licence Text

Normalize | https://github.com/necolas/normalize.css/blob/master/LICENSE.md
---------------------------------
Actual license text